esp/src/eclwatch/templates/HPCCPlatformWidget.html

```diff
@@ -12,10 +12,10 @@
                 </form>
                 <div class="seperator grey"></div>
                 <div id="userAccount">
-                    <span class="navBarLoggedin">${i18n.LoggedInAs}:  </span>
+                    <span class="navBarLoggedin">${i18n.LoggedInAs}: </span>
                     <a id="${id}UserID" href="#" data-dojo-attach-event="onClick:_onUserID"></a>
                     <span id="UserDivider"></span>
-                    <a id="Lock" href="#" data-dojo-attach-event="onClick:_onLock"/></a>
```

Comment:
This looks like an actual change (though I think it was wrong before) rather than just a reformatting?

Reply:
Yes I "fixed" the few reported errors which were caught by the updated formatter (and tested those pages).

Reply:
(each one was a manual change - not automatic).